modpack_format/update_format.md

```diff
@@ -0,0 +1,32 @@
+# Update Format
+The Update Format is a simple format that can be used to specify all available versions of a mod or modpack. It's most useful in the `updates` field in the modpack format, which allows launchers to let users update modpacks automatically.
+
+## Fields
+
+### `formatType`
+Set to `modipUpdates` (TODO: Change name?)
+
+---
+
+### `formatVersion`
+Set to `1.0.0`.
+
+---
+
+### `id`, `name`, `summary`, `description`
```

Comment:
Then why do they need to be repeated here?

Reply:
They are repeated here just so it's clear that these fields exist. This isn't the exact same as the modpack format, so I think it needs to be made clear.

Reply:
I mean why do they need to be in the update checker file as well as the actual modpack? Can they be different?

Reply:
I've put it in the update checker file so you're able to request the update checker *first* - say a mod hosting website servers modpacks, their API could return this update checker file.

Reply:
Is there a reason the update checker file should be the same as the search result API file?

Reply:
There's no specific reason, no, but it's convenient to have one format instead of two.

Reply:
It's also inconvenient to merge unrelated things together. Like the program that tries to be a web browser and also read email.